packages/types/src/ui/stores/preview.ts

```diff
@@ -1,2 +1 @@
 export type PreviewDevice = "desktop" | "tablet" | "mobile"
-export type ComponentContext = Record<string, any>
```

Comment:
This is the other context I was talking about, which is the full app context. This should just be `Record<string, any>`.